src/towncrier/test/test_builder.py

```diff
@@ -81,3 +81,29 @@ def test_strip_with_counter(self):
             parse_newfragment_basename("  007.feature.3", ["feature"]),
             ("7", "feature", 3),
         )
+
+    def test_orphan(self):
```

Comment:
many thanks for the new tests.
I think they are very useful.

I think that for the new tests we should follow best practice and alwasy add a docstring explaining the purpose/scope/scenario of the test.

Can you please add docstring to these new tests? Thanks